config/locales/en/settings.en.yml

```diff
@@ -15,4 +15,9 @@ en:
       disconnect_message: "Your QuickBooks account has been successfully disconnected."
       data_reset_message: "Your QuickBooks data cache has been reset."
       data_reset_information: "If you're experiencing issues with QuickBooks data, click Reset to clear Madeline's cached copy and restore it from QuickBooks."
-      link_message: "Your QuickBooks account has been successfully linked. Note: You need to select accounts before you can add transactions."
+      link_message: "Your QuickBooks account has been successfully linked. Note: Once data is imported, you will need to select accounts before you can add transactions."
+    fetch_status:
+      failed: "The latest background job for fetching QuickBooks data failed."
```

Comment:
I think we can replace all "The latest background job for fetching QuickBooks data" strings with "QuickBooks data import", it's not so much about hiding the implementation details from the user but about using the language for the intended consequence

> QuickBooks data import failed
> QuickBooks data import in progress
> QuickBooks data import pending
> QuickBooks data import succeeded

It's a lot more compact and it frees us up to consider new strategies without changing the user experience.